Tests/ApolloTests/CacheKeyForFieldTests.swift

```diff
@@ -1,5 +1,5 @@
 import XCTest
-@testable import Apollo
```

Comment:
I'm in two minds about this change. I left it in as a forcing function of-sorts for anyone who one day thinks `public` might be the wrong choice. I could easily be convinced to revert this line if someone feels strongly that it should remain `@testable`.

Reply:
I'm good with this, though we should probably do some audits of where we're using `@testable` import separate from this.